usbh_midi.h

```diff
@@ -66,6 +66,11 @@ class USBH_MIDI : public USBDeviceConfig
 public:
         uint16_t pid, vid;
         USBH_MIDI(USB *p);
+        operator bool(){
+          return (pUsb->getUsbTaskState()==USB_STATE_RUNNING && isMidiFound);
+        }
+        uint16_t idVendor(){return vid;}
+        uint16_t idProduct(){return pid;}
```

Comment:
@neroroxxx great if you could add some whitespace and use 8 spaces before the return statement, so it follows the coding style of the existing code.